serde_derive/src/de.rs

```diff
@@ -273,6 +274,18 @@ fn borrowed_lifetimes(cont: &Container) -> BorrowedLifetimes {
     }
 }
 
+fn validate_body(cont: &Container, body: Stmts) -> TokenStream {
+    if let Some(validate) = cont.attrs.validate() {
+        quote! {
+            let body = { #body }?;
+            #validate(&body).map_err(_serde::de::Error::custom)?;
```

Comment:
```suggestion
            #validate(&body).map_err(#serde::de::Error::custom)?;
```

this should be using the `cont.attrs.serde_path()`

Reply:
thx! fixed